appveyor.yml

```diff
@@ -1,11 +1,65 @@
 image: Visual Studio 2015
 
+environment:
+  NUGET_API_KEY:
+    secure: Rh/UHYn7HGj8a7LFWo9C1rSA0sHSQNV3wIqj9bvwRR8yLmCXu9JmbNkmkctk2cBn
```

Comment:
@adamchester I could create an issue with specs and describe the whole feature I'm trying to implement. Will you be interested in code review and traking things up then? Do you have currently time for that? :)